tests/consensus/test_soft_voided3.py

```diff
@@ -18,7 +19,7 @@ def assertNoParentsAreSoftVoided(self, tx):
             self.assertNotIn(settings.SOFT_VOIDED_ID, tx2_voided_by)
 
     def test_soft_voided(self):
-        txA_hash = bytes.fromhex('4586c5428e8d666ea59684c1cd9286d2b9d9e89b4939207db47412eeaabc48b2')
+        txA_hash = bytes.fromhex('1ae4ce163495279dafddca041b7c99abd71af55f29568746f3a20deead15f14d')
```

Comment:
The same here. I can't see txA in both pdfs. Why is that?

Reply:
I think in this case I may have generated the pdfs before adding the labels in the code, since the labels didn't exist before.

I'll generate new ones with the labels.

Reply:
Thanks!